rpcs3/rpcs3qt/rsx_debugger.cpp

```diff
@@ -299,6 +372,35 @@ void Buffer::ShowWindowed() const
 	//}
 }
 
+void Buffer::ShowContextMenu(const QPoint& pos)
+{
+	if (m_image.isNull())
+		return;
+
+	QMenu contextMenu(tr("Save Image"), this);
```

Comment:
snake_case.
also, the text here is obsolete. it won't be shown anyway

Reply:
Fixed.